lib/fog/openstack/identity.rb

```diff
@@ -35,14 +39,6 @@ def initialize(options = {})
           @openstack_identity_prefix = options[:openstack_identity_prefix]
```

Comment:
That said @openstack_identity_prefix can still be used, although I don't have a use case handy.
Although it's not tested so that's bad!
To remove it would require a deprecation approach as it's used in many file